app/controllers/categories_controller.rb

```diff
@@ -163,6 +163,7 @@ def update
         end
       end
 
+      DiscourseEvent.trigger(:category_updated, cat) if result
```

Comment:
This is trivial but you could move this statement up into the `if` right above and not need that trailing `if result`.

Reply:
Done! I added a newline under because I like new lines around `if`s. LMK if you want that removed toooooooo